jobs/v4/job_search_autocomplete_job_title.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
```

Comment:
The directory name says `v4`, but the sample code is using `talent_v4beta1`. Does it make more sense when they match?

Reply:
> I reviewed until the 12nd file, but I think there are many similar issues throughout the sample.
> 
> Can you fix what I pointed out throughout the files?

OK got it